src/urllib3/connectionpool.py

```diff
@@ -768,13 +769,7 @@ def _is_ssl_error_message_from_http_proxy(ssl_error):
                 and self.proxy
                 and _is_ssl_error_message_from_http_proxy(e)
             ):
-                e = ProxyError(
-                    "Your proxy appears to only use HTTP and not HTTPS, "
-                    "try changing your proxy URL to be HTTP. See: "
-                    "https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html"
-                    "#https-proxy-error-http-proxy",
-                    SSLError(e),
-                )
+                e = _wrap_proxy_error(e, conn.proxy)
```

Comment:
Instead of using an empty string for the message let's only wrap the `SSLError` with a `ProxyError` if `conn.proxy.scheme == 'https'`?

Reply:
What do you mean by `let's only wrap the SSLError with a ProxyError`?
if we do it like `ProxyError(SSLError(err))`, It will return error because `ProxyError` needs `message` arg.

Also, I think you meant `conn.proxy.scheme != 'https'`

Reply:
I'm suggesting adding the `conn.proxy.scheme == 'https'` to the `if` condition starting on line 767. That way we only wrap with `ProxyError` with our fancy message if the proxy is HTTPS.

Reply:
Ok, I got it now!
I've added `_wrap_proxy_error` to make it similar to the one that we have on `main` and also easy to test.
As we have [test_https_proxymanager_connected_to_http_proxy](https://github.com/urllib3/urllib3/blob/9397c61c7f447d466487ee8a4b6b2a44cbff0366/test/with_dummyserver/test_socketlevel.py#L1165) I think the changing the `if` condition make sense.